src/utils/storage/index.ts

```diff
@@ -83,13 +85,15 @@ const StorageUtils = {
   otherProfileData: storageBuilder(StorageKeysEnum.OtherProfileData),
   lastPromptNotification: storageBuilder(StorageKeysEnum.LastPromptNotification),
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
+  blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
+  blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
   clearAll: () => MMKVStorage.clearAll()
 };
 
 const clearAll = () => {
   const onboardingPassword = StorageUtils.onboardingPassword.get();
   MMKVStorage.clearAll();
-  StorageUtils.onboardingPassword.set(onboardingPassword);
+  StorageUtils.onboardingPassword.set(onboardingPassword || '');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The modification to the `clearAll` function to set an empty string as the value for `onboardingPassword` if it was previously undefined is a good way to ensure that this key always has a value. However, it's important to consider whether an empty string is an appropriate default value in all contexts where `onboardingPassword` is used.

```diff
   const onboardingPassword = StorageUtils.onboardingPassword.get();
   MMKVStorage.clearAll();
-  StorageUtils.onboardingPassword.set(onboardingPassword);
+  StorageUtils.onboardingPassword.set(onboardingPassword || '');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->